src/main/java/colby/TaskList.java

```diff
@@ -0,0 +1,76 @@
+package colby;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Contains methods to handle the items in the list
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+    public TaskList(List<Task> tasks) {
+        this.tasks = new ArrayList<>(tasks);
+    }
+
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Adds a task to the list
+     * @param task task that will be added to the list
+     */
+    public void addTask(Task task) {
```

Comment:
I really like how the methods follows the naming conventions, starting with a verb